*** tests/A-009-009-M-ok.out	2023-10-06 12:23:29.000000000 +0100
--- tests/A-009-009-M-ok.outhyp	2023-10-27 14:11:45.927987338 +0100
***************
*** 5,15 ****
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
- Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: Menu Principal
- 1 - Novo
- 2 - Abrir
- 3 - Guardar
- 4 - Menu de Edição
- 5 - Menu de Consultas
- 0 - Sair
  Escolha uma opção: 
\ No newline at end of file
--- 5,8 ----
