*** tests/A-015-015-M-ok.out	2023-10-06 12:23:45.000000000 +0100
--- tests/A-015-015-M-ok.outhyp	2023-10-27 19:48:39.315765091 +0100
***************
*** 14,20 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 3;3|#VALUE=4;2
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 14,21 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 3;3|=4;2
! 
  Menu de Edição
  1 - Visualizar
  2 - Inserir
