*** tests/A-026-026-M-ok.out	2023-10-19 17:08:13.000000000 +0100
--- tests/A-026-026-M-ok.outhyp	2023-10-27 19:48:40.724451390 +0100
***************
*** 25,31 ****
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
! Escolha uma opção: Valor a procurar: 1;2|49
  Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
--- 25,35 ----
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
! Escolha uma opção: Menu de Consultas
! 1 - Procurar valores
! 2 - Procurar funções
! 0 - Sair
! Escolha uma opção: Opção inválida!
  Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
