*** tests/A-036-036-M-ok.out	2023-10-19 17:08:13.000000000 +0100
--- tests/A-036-036-M-ok.outhyp	2023-10-27 19:48:44.142116446 +0100
***************
*** 14,20 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 3;1|15=AVERAGE(1;1:2;1)
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 14,23 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 3;1|#VALUE=AVERAGE(1;1|5
! 2;1|25
! )
! 
  Menu de Edição
  1 - Visualizar
  2 - Inserir
