*** tests/A-046-046-M-ok.out	2023-10-19 17:08:13.000000000 +0100
--- tests/A-046-046-M-ok.outhyp	2023-10-27 19:48:47.323666468 +0100
***************
*** 14,45 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): Insira o conteúdo da célula: Menu de Edição
! 1 - Visualizar
! 2 - Inserir
! 3 - Copiar
! 4 - Apagar
! 5 - Cortar
! 6 - Colar
! 7 - Mostrar 'cut buffer'
! 0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 4;1|23=SUB(2;1,2)
! 4;2|23=SUB(2;1,2)
! 4;3|23=SUB(2;1,2)
! Menu de Edição
! 1 - Visualizar
! 2 - Inserir
! 3 - Copiar
! 4 - Apagar
! 5 - Cortar
! 6 - Colar
! 7 - Mostrar 'cut buffer'
! 0 - Sair
! Escolha uma opção: Menu Principal
! 1 - Novo
! 2 - Abrir
! 3 - Guardar
! 4 - Menu de Edição
! 5 - Menu de Consultas
! 0 - Sair
! Escolha uma opção: 
\ No newline at end of file
--- 14,17 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): Insira o conteúdo da célula: 
\ No newline at end of file
