*** tests/A-048-048-M-ok.out	2023-10-19 17:08:13.000000000 +0100
--- tests/A-048-048-M-ok.outhyp	2023-10-27 19:48:48.312148047 +0100
***************
*** 23,29 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 4;2|10=2;3
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 23,30 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 4;2|=2;3
! 
  Menu de Edição
  1 - Visualizar
  2 - Inserir
