*** tests/A-066-066-M-ok.out	2023-10-19 17:08:13.000000000 +0100
--- tests/A-066-066-M-ok.outhyp	2023-10-27 19:48:51.197553791 +0100
***************
*** 23,29 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 1;2|
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 23,30 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 
! 
  Menu de Edição
  1 - Visualizar
  2 - Inserir
***************
*** 33,39 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: 1;1|49
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 34,40 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: xxl.core.CutBuffer@3cd1a2f1
  Menu de Edição
  1 - Visualizar
  2 - Inserir
