*** tests/A-076-076-M-ok.out	2023-10-19 17:17:15.000000000 +0100
--- tests/A-076-076-M-ok.outhyp	2023-10-27 14:03:36.160834817 +0100
***************
*** 2,38 ****
  1 - Novo
  2 - Abrir
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Menu Principal
  1 - Novo
  2 - Abrir
- 3 - Guardar
- 4 - Menu de Edição
- 5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: Menu de Edição
! 1 - Visualizar
! 2 - Inserir
! 3 - Copiar
! 4 - Apagar
! 5 - Cortar
! 6 - Colar
! 7 - Mostrar 'cut buffer'
! 0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 1;1|5
! Menu de Edição
! 1 - Visualizar
! 2 - Inserir
! 3 - Copiar
! 4 - Apagar
! 5 - Cortar
! 6 - Colar
! 7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Menu Principal
  1 - Novo
  2 - Abrir
  3 - Guardar
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: 
\ No newline at end of file
--- 2,22 ----
  1 - Novo
  2 - Abrir
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Abrir: Operação inválida: Problema ao abrir ficheiro: saved27
! Menu Principal
  1 - Novo
  2 - Abrir
  0 - Sair
! Escolha uma opção: Opção inválida!
! Menu Principal
! 1 - Novo
! 2 - Abrir
  0 - Sair
! Escolha uma opção: Especifique o número de linhas da folha: Especifique o número de linhas da folha: Especifique o número de colunas da folha: Menu Principal
  1 - Novo
  2 - Abrir
  3 - Guardar
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: Fim de entrada de dados (R-EOF): pt.tecnico.uilib.text.RuntimeEOFException: Fim do fluxo de dados de entrada
