*** tests/A-091-091-M-ok.out	2023-10-19 17:08:13.000000000 +0100
--- tests/A-091-091-M-ok.outhyp	2023-10-27 19:48:55.790368272 +0100
***************
*** 9,15 ****
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
! Escolha uma opção: Função a procurar: Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
--- 9,17 ----
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
! Escolha uma opção: Função a procurar: 1;1|15
! 
! Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
